site/usage.md

```diff
@@ -0,0 +1,28 @@
+# GetEnvoy CLI Overview
+To run Envoy, execute `getenvoy run -c your_envoy_config.yaml`. This
+downloads and installs the latest version of Envoy for you.
+
+To list versions of Envoy you can use, execute `getenvoy versions -a`. To
+choose one, invoke `getenvoy use 1.18.3`. This installs into
```

Comment:
note the version here is intentionally static, as it is linked directly by our README. Also, our site will pull it in, and it is also static. Changes to the versions list will also trip the unit test of this file, so we don't need to worry about accidental drift.